packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -408,12 +516,17 @@ export const toStyles = ( tree, blockSelectors, hasBlockGapSupport ) => {
 		'.wp-site-blocks > .aligncenter { justify-content: center; margin-left: auto; margin-right: auto; }';
 
 	if ( hasBlockGapSupport ) {
+		// TODO: How do we correctly support different fallback values?
+		const gapValue = getGapCSSValue(
+			tree?.styles?.spacing?.blockGap,
+			'0.5em'
```

Comment:
Is this an arbitrary fallback value? Would this be a good place to use `--wp--style--block-gap` instead, or are we aiming to get rid of that variable altogether?

Reply:
Good catch, thank you, I forgot about that `TODO`. I've been trying to reduce the reliance on the CSS variable to nearly zero in this PR but using `--wp--style--block-gap` here is probably a good pragmatic step, we can always tidy it up further in follow-ups. (This PR still outputs the root blockGap CSS variable since there are other blocks like Gallery that still use it)

Reply:
I think I'll need to dig into this one a little further. In effect, we'll always have a value for `tree?.styles?.spacing?.blockGap` because the root `theme.json` file has a `blockGap` value set (where the `--wp--style--block-gap` value comes from). So, it'd probably be fine if there was no fallback, but safer to have a hard-coded one (and `0.5em` has been used as a hard-coded value elsewhere) 🤔

Separately, I notice that the behaviour in the site editor versus site frontend is slightly different when `settings.spacing.blockGap` is set to `null`, so I'll dig into that further tomorrow, which should hopefully illuminate which setting to use as a fallback here. I hope 😄